### PR TITLE
Transform eject icon into action icon

### PR DESCRIPTION
**Pull request by naaando**
Make eject icon column support any icon
Changes column name from EJECT_ICON to ACTION_ICON

### CODE REVIEW

File: src/View/Sidebar.vala [hunk 1]
```diff
@@ -223,15 +223,15 @@ namespace Marlin.Places {
                                 "active", Column.SHOW_SPINNER,
                                 "pulse", Column.SPINNER_PULSE);
 
-            crpb = new Gtk.CellRendererPixbuf (); /* Icon for eject button (hidden while ejecting or unmounted) */
+            crpb = new Gtk.CellRendererPixbuf (); /* Icon for eject button  (hidden while ejecting or unmounted) and another signs */
```
Comment (jeremypw) [inline src/View/Sidebar.vala]: Double space before bracket in comment. "and another signs" should be "or other icons". Overlong line.  Could probably lose the words in brackets.

File: src/View/Sidebar.vala [hunk 2]
```diff
@@ -223,15 +223,15 @@ namespace Marlin.Places {
...
             this.eject_spinner_cell_renderer = crpb;
             crpb.stock_size = Gtk.IconSize.MENU;
-            crpb.gicon = new ThemedIcon.with_default_fallbacks ("media-eject-symbolic");
             crpb.xpad = ICON_XPAD;
             crpb.ypad = BOOKMARK_YPAD;
 
             col.pack_start (crpb, false);
-            col.set_attributes (crpb, "visible", Column.SHOW_EJECT);
+            col.set_attributes (crpb,
+                                "gicon", Column.ACTION_ICON);
```
Comment (jeremypw) [inline src/View/Sidebar.vala]: Not necessary to split line here.

File: src/View/Sidebar.vala [hunk 3]
```diff
@@ -2110,6 +2109,7 @@ namespace Marlin.Places {
             var rowref = new Gtk.TreeRowReference (store, path);
             store.@set (iter, Column.SHOW_SPINNER, true);
             store.@set (iter, Column.SHOW_EJECT, false);
+            store.@set (iter, Column.ACTION_ICON, null);
```
Comment (jeremypw) [inline src/View/Sidebar.vala]: Not sure why this is necessary.  For this iter, an eject action icon should be appropriate (even if hidden).  It gets set back to an eject icon in `finish_eject_or_unmount ()` anyway.

Comment (naaando) [reply src/View/Sidebar.vala]: It avoids the eject icon to be displayed aside the spinner, when the eject is finished it sets it back if eject has failed.

Comment (jeremypw) [reply src/View/Sidebar.vala]: Ah, OK.  I was just looking at the code - I'll test function asap.

**Pull request merged**